resources/en/translation.json

```diff
@@ -558,6 +558,7 @@
         "error": "Member not found",
         "losses": "Losses",
         "rating": "Rating",
+        "record": "Match Record",
```

Comment:
if we just make sure to keep every item in alphabetical order

Reply:
Will note this for future reference.